src/analyses/libraryFunctions.ml

```diff
@@ -80,6 +80,7 @@ type categories = [
   | `Unlock
   | `ThreadCreate of exp * exp * exp (* id * f  * x       *)
   | `ThreadJoin   of exp * exp (* id * ret_var *)
+  | `MathH        of exp list
```

Comment:
Strictly speaking, the `__builtins` do not belong to any header. Maybe you can just leaf out the `H` because `Math` on its own seems to me like a good category as well

Reply:
Implemented